src/components/UserListItemCard/UserListItem.graphql

```diff
@@ -0,0 +1,10 @@
+fragment UserListItem on User {
+  id
+  fullName
+  displayFirstName {
+    value
+  }
+  displayLastName {
+    value
+  }
```

Comment:
These two properties can be removed right since they are not used?

Reply:
That depends on how we decide to sort, right? Currently we're sorting by these values, but if you and Michael want to go with `realFirstName` and `realLastName` instead we'd need to switch to that

Reply:
Sorting is entirely separate from the returned props. I don't your sorting generic uses a generated TS interface either (which it doesn't need to).